sysutils/starship/Portfile

```diff
@@ -29,8 +29,8 @@ destroot {
 
 cargo.crates \
     aho-corasick                     0.7.6  58fb5e95d83b38284460a5fda7d6470aa0b8844d283a0b614b8535e880800d2d \
-    ansi_term                       0.12.1  d52a9bb7ec0cf484c551830a7ce27bd20d67eac647e1befb56b0be4ee39a55d2 \
     ansi_term                       0.11.0  ee49baf6cb617b853aa8d93bf420db2383fab46d314482ca2803b40d5fde979b \
+    ansi_term                       0.12.1  d52a9bb7ec0cf484c551830a7ce27bd20d67eac647e1befb56b0be4ee39a55d2 \
```

Comment:
Looks like the crates are reordered without actual change. How did you generate the list of crates?

Reply:
Directly from the `Cargo.lock` found in the `v0.24.0` branch

Reply:
You should use the [cargo2port](https://github.com/macports/macports-contrib/blob/master/cargo2port/cargo2port.tcl) script from macports-contrib, which will produce the same results as the previous version in this PR, so you can simply discard the cargo.crates changes here.

Reply:
OK.  cargo2port has been applied.